integration/spark/src/main/scala/org/apache/spark/sql/execution/command/management/CarbonInsertFromStageCommand.scala

```diff
@@ -551,3 +569,21 @@ case class CarbonInsertFromStageCommand(
 
   override protected def opName: String = "INSERT STAGE"
 }
+
+object CarbonInsertFromStageCommand {
+
+  val DELETE_FILES_RETRY_TIMES = 3
+
+  val BATCH_FILE_COUNT_KEY = "batch_file_count"
+
+  val BATCH_FILE_COUNT_DEFAULT: String = Integer.MAX_VALUE.toString
+
+  val BATCH_FILE_ORDER_KEY = "batch_file_order"
+
+  val BATCH_FILE_ORDER_ASC = "ASC"
+
+  val BATCH_FILE_ORDER_DESC = "DESC"
```

Comment:
Add comment to explain that use this option will insert the latest stage files into the table

Reply:
added